cmd/dfdaemon/app/root.go

```diff
@@ -203,6 +203,10 @@ func getConfigFromViper(cmd *cobra.Command, v *viper.Viper) (*config.Properties,
 		cfg.DFRepo = filepath.Join(cfg.WorkHome, "dfdaemon/data/")
 	}
 
+	if cfg.DefaultPattern == "" {
```

Comment:
It comes to me naturally this should be written in a different way:

```golang
const DefaultPattern = "p2p"

..
    if cfg.Pattern == "" {
        cfg.Pattern = DefaultPattern
    }
```

I'm uncertain we should call it `DefaultPattern` in the configuration type. Could we just call it `Patter` or whatever fit?

Reply:
Done.